fighthealthinsurance/ml/ml_router.py

```
@@ -67,9 +67,7 @@ def entity_extract_backends(self, use_external) -> list[RemoteModelLike]:
         else:
             return self.internal_models_by_cost
 
-    def summarize(
-        self, text: Optional[str], query: str, abstract: Optional[str]
-    ) -> Optional[str]:
+    def summarize(self, text: Optional[str], abstract: Optional[str]) -> Optional[str]:
```

Ruff (0.8.2)

`Optional` may be undefined, or defined from star imports

(F405)